src/Catty/DataModel/Bricks/Look/SetBackgroundByIndexBrick.h

```diff
@@ -0,0 +1,36 @@
+/**
+ *  Copyright (C) 2010-2021 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+#import "Brick.h"
+#import "SpriteObject.h"
+#import "BrickLookProtocol.h"
+#import "BrickFormulaProtocol.h"
+
+@class Formula;
+
+@interface SetBackgroundByIndexBrick : Brick<BrickProtocol>
+
+@property (nonatomic, strong) Formula *backgroundIndex;
```

Comment:
Please do not write new Bricks in Objective C. Our midterm goal is to only have Swift code and get rid of almost all Objective C code. For further guidance, please have a look at `SetLookByIndexBrick.swift`.